src/components/tooltip/index.tsx

```diff
@@ -7,9 +7,24 @@ type Props = {
 } & Pick<TooltipProps, 'label' | 'placement' | 'maxWidth'>;
 
 const Tooltip: FC<Props> = ({ children, ...props }) => {
+  const [isLabelOpen, setIsLabelOpen] = useState<boolean>(false);
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const childrenWithProps = cloneElement(children as React.ReactElement<any>, {
```

Comment:
Would it work to specify props types instead of `any`?
Something like this:

```suggestion
  type EventHandlers = {
    onClick: () => void;
    onMouseEnter: () => void;
    onMouseLeave: () => void;
  };

  const childrenWithProps = cloneElement(children as React.ReactElement<EventHandlers>, {
```

Reply:
thanks